docs/content/s3.md

```diff
@@ -6040,3 +6041,121 @@ d) Delete this remote
+
+First run:
+
+```
+rclone.exe config
+```
+
+```
+This will guide you through an interactive setup process:
+No remotes found, make a new one?
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+
+Enter name for new remote.
+name>My zata storage
+
+Type of storage to configure.
+Choose a number from below, or type in your own value.
+XX / Amazon S3 Compliant Storage Providers including AWS,(included Zata also) ...
+\ (s3)
+Storage> 4 (Press with the matching number) Or s3
+
+Choose your S3 provider.
+Choose a number from below, or type in your own value.
+36 / Zata (S3 compatible Gateway)
+\ (Zata)
+provider> Zata
+
+Get AWS credentials from runtime (environment variables or EC2/ECS meta data if no env vars).
+Only applies if access_key_id and secret_access_key is blank.
+Choose a number from below, or type in your own boolean value (true or false).
+Press Enter for the default (false).
+1 / Enter AWS credentials in the next step.
+\ (false)
+2 / Get AWS credentials from the environment (env vars or IAM).
+\ (true)
+env_auth> 1
+
+AWS Access Key ID.
+Leave blank for anonymous access or runtime credentials.
+Enter a string value. Press Enter for the default ("").
+access_key_id> "Your access Key ID"
+
+AWS Secret Access Key (password).
+Leave blank for anonymous access or runtime credentials.
+Enter a string value. Press Enter for the default ("").
+secret_access_key> "Your secret access Key ID"
+
+Region where you can connect with.
+Choose a number from below, or type in your own value.
+Press Enter to leave empty.
+1 / Indore, Madhya Pradesh, India
+\ (us-east-1)
+region> 1
+
+Endpoint for Zata Object Storage.
+Choose a number from below, or type in your own value.
+Press Enter to leave empty.
+1 / South Asia Endpoint
+\ (idr01.zata.ai)
+endpoint> 1
+
+Endpoint for S3 API.
+Required when using an S3 clone.
+Enter a value of type string. Press Enter for the default (idr01.zata.ai).
+endpoint>
+
+Location constraint - must be set to match the Region.
+Leave blank if not sure. Used when creating buckets only.
+Enter a value. Press Enter to leave empty.
+location_constraint>
+
+Canned ACL used when creating buckets and storing or copying objects.
+Choose a number from below, or type in your own value.
+Press Enter to leave empty.
+1 / Owner gets FULL_CONTROL. No one else has access rights (default).
+\ (private)
+2 / Owner gets FULL_CONTROL. The AllUsers group gets READ access.
+\ (public-read)
+3 / Owner gets FULL_CONTROL. The AllUsers group gets READ and WRITE access.
+\ (public-read-write)
+4 / Owner gets FULL_CONTROL. The AuthenticatedUsers group gets READ access.
+\ (authenticated-read)
+5 / Object owner gets FULL_CONTROL. Bucket owner gets READ access.
+\ (bucket-owner-read)
+6 / Both the object owner and the bucket owner get FULL_CONTROL over the object.
+\ (bucket-owner-full-control)
+acl> 1
+
+Edit advanced config?
+y) Yes
+n) No (default)
+y/n> n
+
+Configuration complete.
+Options:
+
+type: s3
```

Comment:
Please cut and paste exactly what a trial config session says here (see other examples in this file)